virtool_core/models/enums.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class Permissions(Enum):
```

Comment:
Conventional to name enum in the singular. See examples in `enum` docs.

```suggestion
class Permission(Enum):
```